text/src/autogluon/text/automm/predictor.py

```diff
@@ -147,6 +147,8 @@ def __init__(
             eval_metric = eval_metric.name
 
         if eval_metric is not None and eval_metric.lower() in ["rmse", "r2", "pearsonr", "spearmanr"]:
+            if problem_type == BINARY:
+                bce_regression_state = True
```

Comment:
Cannot understand the logic.

Reply:
In my original design, this mode is determine to introduce binary into regression. So users treat it as a binary problem (but regression indeed) and the eval metrics may be regression.